js/basicblocks.js

```diff
@@ -13,7 +13,7 @@
 
 // Some names changed between the Python verison and the
 // JS version so look up name in the conversion dictionary.
-var NAMEDICT = {
+let NAMEDICT = {
```

Comment:
const can be used here and other relevant variales where mutation is not recommended